src/components/duelling-picklist/picklist-placeholder/picklist-placeholder.component.tsx

```diff
@@ -1,17 +1,16 @@
 import React from "react";
-import PropTypes from "prop-types";
 
 import { StyledPicklistPlaceholder } from "../duelling-picklist.style";
 
-const PicklistPlaceholder = ({ text }) => (
+export interface PicklistPlaceholderProps {
+  /** Text to be displayed when list is empty */
+  text: string;
+}
+
+export const PicklistPlaceholder = ({ text }: PicklistPlaceholderProps) => (
   <StyledPicklistPlaceholder data-element="picklist-placeholder">
     {text}
   </StyledPicklistPlaceholder>
 );
 
-PicklistPlaceholder.propTypes = {
-  /** Text to be displayed when list is empty */
-  text: PropTypes.string.isRequired,
-};
-
 export default PicklistPlaceholder;
```

Comment:
```suggestion
export default PicklistPlaceholder;
PicklistPlaceholder.displayName = "PicklistPlaceholder";
```

Reply:
There's no need for the displayName assignment here as the PicklistPlaceholder is not wrapped in forwardRef